### PR TITLE
Add tests for BootstrapControlPlane.

### PR DESCRIPTION
**Pull request by diegs**

### CODE REVIEW

**Comment by diegs** [comment]
rktbot run tests

File: pkg/bootkube/bootstrap_test.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+		t.Error("bcp.Teardown() failed to delete secrets directory")
+	}
+	for _, manifest := range manifests {
+		if fi, err := os.Stat(filepath.Join(podManifestPath, manifest)); fi != nil || !os.IsNotExist(err) {
+			t.Errorf("bcp.Teardown() failed to delete manifest: %v", manifest)
+		}
+	}
+}
+
+func TestBootstrapControlPlane_NoOverwrite(t *testing.T) {
```
Comment (yifan-gu) [inline pkg/bootkube/bootstrap_test.go]: small nit: usually function name doesn't have underscore in go

**Comment by yifan-gu** [comment]
LGTM other than one small nit.

**Comment by yifan-gu** [comment]
@diegs Thanks! LGTM after squash :)

**Comment by diegs** [comment]
@yifan-gu thanks! But you didn't (or can't?) lgtm :)

**Pull request merged**